litellm/llms/bedrock/chat/converse_transformation.py

```diff
@@ -344,10 +347,15 @@ def _transform_request(
                 additional_request_keys.append(k)
         for key in additional_request_keys:
             inference_params.pop(key, None)
+            # Add Support for Amazon Nova topK: https://docs.aws.amazon.com/nova/latest/userguide/complete-request-schema.html
+            if key == "top_k":
```

Comment:
there is no need for this if condition. Line 346 should handle this.

Reply:
please add the topk mapping here - https://github.com/BerriAI/litellm/blob/a0aef3b53a015d4877cd91d74edad468949d4f5e/litellm/llms/bedrock/chat/converse_transformation.py#L230

Reply:
I tried using map_openai_params and I couldn't get it to move under inferenceConfig object in additionalRequestParams

Let me take another look

Reply:
> there is no need for this if condition. Line 346 should handle this. 

The topK has to move under inferenceConfig key

I suppose we could move the condition into the block above

Reply:
> please add the topk mapping here inside `map_openai_params()` - https://github.com/BerriAI/litellm/blob/a0aef3b53a015d4877cd91d74edad468949d4f5e/litellm/llms/bedrock/chat/converse_transformation.py#L230

I tried using map_openai_params and I couldn't get it to move under inferenceConfig object in additionalRequestParams

Let me take another look

Reply:
@krrishdholakia I tested your PR and it isn't setting `topK` in the place Bedrock Nova expects, added sample test to repro to the original report here:

https://github.com/BerriAI/litellm/issues/7087#issuecomment-2543378690